react-with-local-storage/react-with-local-storage_stored-note/src/App.jsx

```diff
@@ -1,11 +1,11 @@
-import { useState } from "react";
+import useLocalStorageState from "use-local-storage-state";
 import FontSelector from "./components/FontSelector";
 import Note from "./components/Note";
 import "./App.css";
 
 export default function App() {
-  const [note, setNote] = useState("");
-  const [font, setFont] = useState("system-ui");
+  const [note, setNote] = useLocalStorageState("note", { defaultValue: [] });
+  const [font, setFont] = useLocalStorageState("font", { defaultValue: [] });
```

Comment:
I guess, this line should be like this:

```javascript 
const [font, setFont] = useLocalStorageState("font", {
    defaultValue: "system-ui",
});
```

... since this line looked like this before the refactoring:

```javascript
const [font, setFont] = useState("system-ui");
```